README.md

```diff
@@ -1,3 +1,9 @@
+⚠️ ⚠️ ⚠️
+
+WebAuthn-json has been sunset. Now that [all major browsers support WebAuthn](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API#browser_compatibility) we recommend invoking the native APIs
```

Comment:
What do you think of linking https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API ?